static/javascripts/components/survey_card.jsx

```diff
@@ -67,9 +93,8 @@ var SurveysPage = React.createClass({
             item_title: this.state.item_title,
```

Comment:
Ah I see, but I'm a little confused. In the getinitialstate() function, we initialize a field called "item_name". Isn't "item_name" the actual title of the survey? Then you have some function called updateTitle(), and this updates a new field called "item_title". Shouldn't that be "item_name" instead?